hub/api/dataset.py

```diff
@@ -5,15 +5,13 @@
 import numpy as np
```

Comment:
oh one more thing: please move the `Dataset` class file into `core`. the dataset handler can remain in API. 

maybe keep tensor in API for now? it would also make sense to move this to the core but for now there's no reason to